tests/test_vector2_angle.py

```diff
@@ -29,6 +29,7 @@ def test_angle(left, right, expected):
     right=vectors(),
 )
 def test_angle_range(left, right):
+    """Vector2.angle produces values in [-180; 180] and is antisymmetric."""
```

Comment:
Is antisymmetric different from asymmetric?

Reply:
Yes.  Antisymmetric means (in that case) that `left.angle(right) == - right.angle(left)`: it's symmetric, but with a twist (a sign change).
I will clarify that.

Reply:
BTW, I considered changing those vector names to `x` and `y`, since `left` and `right` aren't particularly meaningful there.

Reply:
Is dc0f688 enough clarification on antisymmetry?
Do you have an opinion on the variable name change I suggested?

Reply:
Names seem reasonable, or `vector_a` and `vector_b`.

Ehhh.... defining it doesn't actually feel better?

Reply:
What would you suggest then?  Having a name for the property is useful, IMO (and that's a commonly-accepted name)

Reply:
I meant more that "antisymmetric" doesn't mean much to non-mathematicians.

Simply adding as an additional paragraph that flipping the order flips the sign is more helpful than trying to summarize it in a single word.

Reply:
Yes, and there is a clarification right afterwards:
```
def test_angle_range(left, right):
    """Vector2.angle produces values in [-180; 180] and is antisymmetric.

    Antisymmetry means that left.angle(right) == - right.angle(left).
    """
```